app/plugin.go

```diff
@@ -502,6 +502,7 @@ func (a *App) GetMarketplacePlugins(filter *model.MarketplacePluginFilter) ([]*m
 
 		result = append(result, &model.MarketplacePlugin{
 			BaseMarketplacePlugin: &model.BaseMarketplacePlugin{
+				// Labels should not be localized as the labels send by the Marketplace are neither.
```

Comment:
```suggestion
				// Labels should not (yet) be localized as the labels sent by the Marketplace are not (yet) localizable.
```

Reply:
Should I link the ticket I created for this in the code?

Reply:
Nah, I don't think that's strictly necessary.